arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java

```diff
@@ -397,6 +439,23 @@ private int calc(String val1, String val2) {
     return len1 + len2;
   }
 
+  private int coincidePath(Map<Integer, Long> replayEigenMap, Map<Integer, Long> recordEigenMap) {
+    int row = 0;
+    if (MapUtils.isEmpty(replayEigenMap) || MapUtils.isEmpty(recordEigenMap)) {
+      return row;
+    }
+
+    for (Map.Entry<Integer, Long> entry : recordEigenMap.entrySet()) {
+      Integer key = entry.getKey();
+      Long recordPathValue = recordEigenMap.get(key);
+      Long replayPathValue = replayEigenMap.get(key);
+      if (recordPathValue.equals(replayPathValue)) {
```

Comment:
Using Objects.equals(Object, Object) method for comparison to prevent null pointer exceptions.